content/docs/command-reference/artifacts/get.md

```diff
@@ -0,0 +1,92 @@
+## artifacts get
+
+Download an artifact tracked in a DVC project into the current working
+directory.
+
+## Synopsis
+
+```usage
+usage: dvc artifacts get [-h] [-q | -v]
+                         [--rev [<version>]] [--stage [<stage>]]
+                         [-o [<path>]] [-j <number>] [-f]
+                         [--config CONFIG]
+                         [--remote REMOTE] [--remote-config [REMOTE_CONFIG ...]]
+                         url name
+
+positional arguments:
+  url                   Location of DVC repository to download from
+  name                  Name of artifact in the repository
+```
+
+## Description
+
+Provides an easy way to download artifacts tracked in a DVC project.
+`dvc artifacts get` supports downloading artifacts both from the
+[Studio Model Registry](/doc/studio/user-guide/model-registry) and from DVC
+remotes. Unlike `dvc get`, `dvc artifacts get` supports downloading an artifact
+by name, rather than by path.
+
+<admon type="tip">
+
+Downloading an artifact from the Studio Model Registry only requires a valid
+Studio
+[access token](/doc/studio/user-guide/account-management#studio-access-token).
+It does not require the client to have DVC remote credentials.
+
+</admon>
+
+The `url` argument specifies the address of the DVC or Git repository containing
+the artifact. Both HTTP and SSH protocols are supported (e.g.
+`[user@]server:project.git`). `url` can also be a local file system path
+(including the current project e.g. `.`).
+
+The `name` argument specifies the name of the artifact to download.
+
+<admon type="info">
+
+`dvc artifacts get` will first try to download artifacts from Studio. If you do
```

Comment:
`from Studio` reads a bit like Studio itself is hosting them. Can we change it a bit? explain the benefit of this?

Reply:
This feels like something that should be documented on the Studio side, especially given that the Studio frontend is also going to provide download links using the signed https URLs. (And then the `dvc artifacts get` docs can link the Studio page for details)

related: https://github.com/iterative/dvc.org/pull/4681
(I'm not sure that PR should have been closed?)

Reply:
Let's get #4681 merged and then link it here.

Reply:
@pmrowla Can you set this PR to be on top of #4681 and reference the studio api docs here?

Reply:
ping @pmrowla

Reply:
Let's add it in #4681